coverage.txt

```diff
@@ -265,28 +257,26 @@ github.com/giantswarm/gsctl/commands/create_keypair.go:43.2,43.24 1 1
-github.com/giantswarm/gsctl/commands/create_cluster.go:347.46,353.4 5 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:339.3,347.46 4 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:335.21,338.4 1 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:330.2,330.16 1 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:330.16,335.21 3 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:321.17,327.4 5 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:319.16,321.17 2 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:318.2,319.16 2 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:313.3,314.16 2 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:310.17,312.4 1 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:307.2,307.16 1 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:307.16,310.17 3 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:299.81,303.4 3 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:297.28,299.81 1 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:297.2,297.28 1 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:291.4,291.14 1 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:287.5,290.5 2 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:284.30,287.5 2 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:282.4,284.30 2 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:280.39,282.4 1 0
-github.com/giantswarm/gsctl/commands/create_cluster.go:278.28,280.39 1 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:349.4,357.4 7 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:343.46,349.4 5 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:335.3,343.46 4 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:331.21,334.4 1 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:326.2,326.16 1 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:326.16,331.21 3 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:317.17,323.4 5 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:315.16,317.17 2 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:314.2,315.16 2 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:309.3,310.16 2 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:306.17,308.4 1 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:303.2,303.16 1 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:303.16,306.17 3 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:295.81,299.4 3 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:293.28,295.81 1 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:293.2,293.28 1 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:287.3,287.13 1 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:283.4,286.4 2 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:280.29,283.4 2 0
+github.com/giantswarm/gsctl/commands/create_cluster.go:278.28,280.29 2 0
```

Comment:
It has to be committed, otherwise there can't be test coverage comparison before/after by codecov.